XX.md

```diff
@@ -0,0 +1,31 @@
+NIP-XX
+======
+
+External Content IDs
+-------------------------
+
+`draft` `optional`
+
+There are certain established global content identifiers that would be useful to reference in nostr events so that clients can query all events assosiated with these ids.
+
+- Book [ISBNs](https://en.wikipedia.org/wiki/ISBN)
+- Podcast [GUIDs](https://podcastnamespace.org/tag/guid)
+- Movie [EIDRs](https://www.eidr.org)
+
+Since the `i` tag is already used for similar references in kind-0 metadata events it makes sense to use it for these content ids as well.
+
+
+## Supported IDs
+
+### Books:
+
+- Book ISBN: `["i", "book:isbn:123"]`
+
+### Podcasts:
+
+- Podcast Feed GUID: `["i", "podcast:guid:123"]`
+- Podcast Item GUID: `["i", "podcast:item:guid:123"]`
```

Comment:
Please use actual examples. ISBN and GUID have formats.

I think it would also be required to mention what to do if a standard uses `:` as separator itself like in IPv6: ` 2001:0db8:0000:0000:0000:ff00:0042:8329` is a valid IPv6.

Casing should also be addressed to not get both ISBN and isbn.

```suggestion
- Book ISBN: `["i", "book:isbn:0-201-63361-2"]`

### Podcasts:

- Podcast Feed GUID: `["i", "podcast:guid:396d9ae0-da7e-5557-b894-b606231fa3ea"]`
- Podcast Item GUID: `["i", "podcast:item:guid:396d9ae0-da7e-5557-b894-b606231fa3ea"]`
```

Reply:
@Giszmo thanks for the suggestions I've added real examples for all 3.

I also switched the movie references to use `isan`s rather than `eidr`s because they are simpler and easier to ignore multiple edits of the same movie.

For standards that use `:` in the value my assumption is that's fine as the value is always the last thing in the string - I can add some text about this if you think that's necessary?

Reply:
I think it would be indeed helpful to clarify that : must not get escaped in any way. That should be enough hint for implementers to consider this case.